src/sqlancer/clickhouse/ClickHouseErrors.java

```diff
@@ -62,7 +62,17 @@ public static void addExpectedExpressionErrors(ExpectedErrors errors) {
         errors.add("ILLEGAL_DIVISION");
         errors.add("DECIMAL_OVERFLOW");
         errors.add("Cannot convert out of range floating point value to integer type");
+        errors.add("Unexpected inf or nan to integer conversion");
+        errors.add("No such name in Block::erase"); // https://github.com/ClickHouse/ClickHouse/issues/42769
```

Comment:
For such unfixed issues, would you like to adopt the pattern described in https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md#unfixed-bugs? I think this would make the code more maintainable (let me know if you don't want to).

Reply:
I remember you showed this one already. I usually do not know what should be disabled to disable behavior with errors. But I know that that issue will soon be fixed and I will remove this row